transport.go

```diff
@@ -94,7 +94,6 @@ func addWebhookRequestDecoder(config transportConfig) kithttp.DecodeRequestFunc
 		var (
 			v1       *webhook.RegistrationV1
 			v2       *webhook.RegistrationV2
-			whreq    addWebhookRequest
```

Comment:
unit tests were failing. needed a pointer. removed the initiation of this variable and initialize struct at return instead